src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/DecXtractEventListener.java

```diff
@@ -90,7 +91,7 @@ private void handleDeleteIssue(DecisionKnowledgeElementImpl decisionKnowledgeEle
 	}
 
 	private void handleEditComment(DecisionKnowledgeElementImpl decisionKnowledgeElement) {
-		if(!KnowledgeRest.commentWriteLog) {
+		if(!DecXtractEventListener.editCommentLock) {//If locked, a rest service is manipulating the comment and should not be handled by EL
```

Comment:
Is it really just "the" comment or could this also happen if in parallel another user updates some other comment?

Reply:
Good point. Need to think about that